staging/src/k8s.io/apimachinery/pkg/util/version/version.go

```diff
@@ -333,6 +333,34 @@ func (v *Version) String() string {
 	return buffer.String()
 }
 
+// MajorMinorString returns the version as "major.minor",
+// omitting patch, pre-release, and build metadata.
+// Used for EmulationVersion display in leases and logs.
+func (v *Version) MajorMinorString() string {
```

Comment:
We already have `String()` receiver that will print the string out correctly if provided the correct components.  Let's leverage this.  Maybe allow for `MajorMinor().String()` by adding a `MajorMinor() *Version` receiver? Same for `MajorMinorPatch`.

Reply:
Yeah, reusing the logic in `String()` seems more maintainable.